settings.py

```diff
@@ -31,7 +31,7 @@
     }
     if url.scheme == 'postgres':
         DATABASES['default']['ENGINE'] = 'django.db.backends.postgresql_psycopg2'
-    elif url.scheme == 'mysql':
+    elif url.scheme.startswith('mysql'):
         DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
```

Comment:
This deployed correctly and you can create entries, right?

Reply:
Yup! Although that reminded me that I didn't checked it for postgres.

Reply:
nothing to do